src/caps.rs

```diff
@@ -0,0 +1,48 @@
+use std::time::Duration;
+
+use anyhow::{Context, Result};
+use ed25519_dalek::{SigningKey, VerifyingKey};
+use iroh::NodeId;
+use rcan::{Capability, Expires, Rcan};
+use serde::{Deserialize, Serialize};
+use ssh_key::PrivateKey as SshPrivateKey;
+
+/// Potential capabilities for IPS
+#[derive(Ord, Eq, PartialOrd, PartialEq, Clone, Serialize, Deserialize, Debug)]
```

Comment:
Re: `derive(Serialize)`: Ideally the `IpsCap` [is wrapped with a version](https://discord.com/channels/949724860232392765/1185651743510515712/1333550615552589895).

Maybe we should require a version number in rcan for capabilities after all :upside_down_face:

Reply:
version added